packages/cli/src/cli/provider/action.provider.history.ts

```
@@ -0,0 +1,73 @@
+import { Aws, TileMetadataProviderRecord, TileMetadataTag } from '@basemaps/lambda-shared';
+import { CommandLineAction } from '@rushstack/ts-command-line';
+import { CliTable } from '../cli.table';
+import { printProvider } from './provider.util';
+
+const MaxHistory = 199;
+
+export class ProviderHistoryAction extends CommandLineAction {
+    public constructor() {
+        super({
+            actionName: 'log',
+            summary: 'Show history for the provider details',
+            documentation: '',
+        });
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    protected onDefineParameters(): void {}
+
+    async getAllTags(): Promise<Map<TileMetadataTag, TileMetadataProviderRecord>> {
+        const allTags: Map<TileMetadataTag, TileMetadataProviderRecord> = new Map();
+        await Promise.all(
+            Object.values(TileMetadataTag).map(async (tag) => {
+                try {
+                    const value = await Aws.tileMetadata.Provider.get(tag);
+                    allTags.set(tag, value);
+                } catch (e) {}
```

Do we really want to swallow _all_ errors?